app/services/category_hashtag_data_source.rb

```diff
@@ -22,6 +22,7 @@ def self.category_to_hashtag_item(category)
       item.slug = category.slug
       item.description = category.description_text
       item.icon = icon
+      item.colors = [category.parent_category&.color, category.color].compact
```

Comment:
Need to add this to the chat channel hashtag data source as well.

Reply:
Also need to make sure that this information does not leak anymore. To try this, you can make a private category that a certain user cannot access, and see what happens when you do `#categoryslug` as that user, e.g. if they get the colours and other category info sent to them.